### PR TITLE
add example for docker usage

### PR DESCRIPTION
**Pull request by maryamsulemani97**
Replaces https://github.com/meilisearch/documentation/pull/1602

### CODE REVIEW

File: learn/getting_started/quick_start.md [hunk 1]
```diff
@@ -53,11 +53,13 @@ These commands launch the **latest stable release** of Meilisearch.
...
 docker pull getmeili/meilisearch:latest
 
-# Launch Meilisearch
+# Launch Meilisearch with a master key and deactivate analytics
 docker run -it --rm \
     -p 7700:7700 \
+    -e MEILI_MASTER_KEY='myMasterKey'\
     -v $(pwd)/meili_data:/meili_data \
     getmeili/meilisearch:latest
+    /meilisearch --no-analytics
```
Comment (guimachiavelli) [inline learn/getting_started/quick_start.md:62]: I didn't follow the issue very closely during v0.27, but I thought [our official recommendation](https://github.com/meilisearch/meilisearch/issues/2235) was using `meilisearch` with Docker (so, no `/`)?

```suggestion
    meilisearch --no-analytics
```

**Review by guimachiavelli — APPROVED** [review]
One minor comment regarding the recommended binary path, but LGTM otherwise 🦄

File: learn/getting_started/quick_start.md (hunk 1 above)
Comment (maryamsulemani97) [reply learn/getting_started/quick_start.md:62]: You're right!

File: learn/getting_started/quick_start.md [hunk 2]
```diff
@@ -53,11 +53,13 @@ These commands launch the **latest stable release** of Meilisearch.
...
 docker pull getmeili/meilisearch:v0.27.1
 
-# Launch Meilisearch
+# Launch Meilisearch with a master key and deactivate analytics
 docker run -it --rm \
     -p 7700:7700 \
+    -e MEILI_MASTER_KEY='myMasterKey'\
     -v $(pwd)/meili_data:/meili_data \
     getmeili/meilisearch:v0.27.1
+    meilisearch --no-analytics
```
Comment (dichotommy) [inline learn/getting_started/quick_start.md:62]: This change might result in us losing some data from Docker users. How about we replace `--no-analytics` with `--env="development"`?

Also, have you tested this to be sure that it works?

Comment (maryamsulemani97) [inline learn/getting_started/quick_start.md:62]:
```suggestion
    meilisearch --env="development"
```

File: learn/getting_started/quick_start.md [hunk 3]
```diff
@@ -53,11 +53,13 @@ These commands launch the **latest stable release** of Meilisearch.
 # Fetch the latest version of Meilisearch image from DockerHub
 docker pull getmeili/meilisearch:v0.27.1
 
-# Launch Meilisearch
+# Launch Meilisearch with a master key and deactivate analytics
```
Comment (maryamsulemani97) [inline learn/getting_started/quick_start.md:56]:
```suggestion
# Launch Meilisearch in development mode with a master key
```

File: learn/getting_started/quick_start.md (hunk 2 above)
Comment (maryamsulemani97) [reply learn/getting_started/quick_start.md]: Yes I did

**Review by guimachiavelli — APPROVED** [review]
🐗

**Pull request merged**